app/controllers/concerns/remember_device_concern.rb

```diff
@@ -1,6 +1,10 @@
 module RememberDeviceConcern
   extend ActiveSupport::Concern
 
+  def save_user_opted_remember_device_pref
+    cookies.encrypted[:user_opted_remember_device_preference] ||= params[:remember_device]
```

Comment:
because of the way truthy-falsy values work, doesn't this mean once this is set, it can never be unset?

Do we need to explicityl check for `.key?`

Reply:
Good spot... actually, I think the ||= isn't even necessary here, that might have gotten left behind from an old thought I had. Simply `cookies.encrypted[:user_opted_remember_device_preference] ||= params[:remember_device]` should work, the cookie should be set as whatever is in params, always. Thanks!

Reply:
Anyways it worked with the ||= because we don't really ever need to unset the cookie. It's meant to stick around until the browser clears it

Reply:
Ok sounds good!